scripts/db_access/client_side_db.py

```diff
@@ -227,18 +244,41 @@ def _main(soledad, km, args):
             yield _export_key(args, km, args.expoert_public_key, private=False)
         if args.export_incoming_messages:
             yield _export_incoming_messages(soledad, args.export_incoming_messages)
-    except:
-        pass
+    except Exception as e:
+        logger.error(e)
     finally:
-        reactor.stop()
+        soledad.close()
+        reactor.callWhenRunning(reactor.stop)
```

Comment:
I don't quite understand this. this shouldn't be a callback to any action? this way, we're stopping it as soon as it starts, does that makes sense? maybe reactor.callLater(1, reactor.stop) or something?